pkg/networking/client.go

```diff
@@ -0,0 +1,40 @@
+package networking
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+)
+
+// Client is a networking client for DC/OS.
+type Client struct {
+	http *httpclient.Client
+}
+
+// NewClient creates a new networking client.
+func NewClient(baseClient *httpclient.Client) *Client {
+	return &Client{
+		http: baseClient,
+	}
+}
+
+// Nodes returns the nodes in a cluster.
+func (c *Client) Nodes() (*[]Node, error) {
```

Comment:
There is no need to return a pointer to the slice. See https://github.com/dcos/dcos-core-cli/blob/08bbf476ca0cd7a7126cce5092bec22a86ba4333/pkg/metronome/client.go#L100